react-native-libraries.json

```diff
@@ -10752,5 +10754,10 @@
     "examples": ["https://github.com/bluesky-social/react-native-uitextview/tree/main/example"],
     "images": ["https://haileyok.com/content/images/size/w1920/2024/01/IMG_4730.jpeg"],
     "ios": true
+  },
+  {
+    "githubUrl": "https://github.com/callstack/react-native-windows-hello",
+    "npmPkg": "react-native-windows-hello",
```

Comment:
Hello @Yajur-Grover, thanks for the new addition and updates! 👍 

I have verified that the SVG and Lottie includes a Windows example, since it's a requirement for adding support for out-of-tree platforms. Just have one nit , before we ship this:

```suggestion
```

Since npm package matches repository name we can skip adding `npmPkg` field.